### ISSUE TITLE
[a11y] SVG icons in the docs should have an ALT text, to allow screen readers to access this info

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#7391]
In the TW documentation we do use SVG icons as hints, but those icons don't have an ALT text, because they are transcluded. 

see: https://tiddlywiki.com/#acos%20Operator

**Once #6666 is merged it should be straight forward to give those elements an alt-text** which seems to be considered mandatory according to MDN: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#value

May be it's already possible now. ... We should check this. 

just a reminder.
